### PR TITLE
Turret mount crafting recipe

### PR DESCRIPTION
**Pull request by DziugasPraninskas**
#### Summary

```SUMMARY: Content "Add turret mount crafting recipe"```
#### Purpose of change

Turret mounts should be craftable by the player, seeing as they merely affix the gun to the vehicle.
#### Describe the solution

Add turret mount crafting recipe.
#### Describe alternatives you've considered

The recipe could use different tool qualities or components, depending on game balance or realism.

P.S. First PR, be gentle.

### CODE REVIEW

File: data/json/recipes/recipe_vehicle.json [hunk 1]
```diff
@@ -452,6 +452,22 @@
...
+    "subcategory": "CSC_OTHER_VEHICLE",
+    "skill_used": "mechanics",
+    "difficulty": 2,
+    "time": "30 m",
+    "autolearn": true,
+    "using": [ [ "welding_standard", 3 ] ],
+    "qualities": [ { "id": "HAMMER", "level": 2 }, { "id": "SAW_M", "level": 1 }, { "id": "WRENCH", "level": 1 } ],
+    "components": [
+      [ [ "pipe", 3 ] ],
+      [ [ "steel_chunk", 4 ], [ "scrap", 12 ] ],
```
Comment (Fris0uman) [inline data/json/recipes/recipe_vehicle.json]: You should use "steel_standard" requirement instead.

Comment (Fris0uman) [inline data/json/recipes/recipe_vehicle.json]:
```suggestion
      [ [ "steel_chunk", 4 ], [ "scrap", 12 ] ]
```
No `,` at the end of an array

**Comment by Fris0uman** [comment]
Shouldn't this also include some gear or some mobile part so that the gun can be moved?

**Comment by Fris0uman** [comment]
You need to lin your changes, copy paste it in http://dev.narc.ro/cataclysm/format.html , click `Lint`and copy/paste it back into your file.

**Comment by Dziugaske** [comment]
Alright, thanks for the help Fris.

**Comment by Rail-Runner** [comment]
Since this is a turret mount that can be operated remotely (from vehicle controls), you may want to add electric components to the recipe; 2 tiny electric motors and some copper cables would probably do it (and using those in the recipe would mandate the use of soldering standard requirement in addition to welding, as well as screw driving quality); also it'd need springs (one or more) as at least some attempt of stabilizing the gun. With those additions it'd probably be reasonable to increase difficulty and crafting time as well.

Also you may want to remove the current recipe for this item from Aftershock; I think it comes from the time when all turret mounts were capable of autonomously finding and shooting at a target.

Also make sure that the components used weigh about as much as the finished item; for reference, turret mount weighs 10 kg currently, but the weight of three pipes currently used for making it is only about 3.75 kg.

**Comment by Dziugaske** [comment]
The turret mount recipe now requires motors to operate it remotely, thanks for the suggestion. Also removed the turret mount recipe from Aftershock.

**Comment by I-am-Erk** [comment]
I don't think three pipes alone represent enough structural stability. I'd add a few steel lumps at least.

**Comment by Dziugaske** [comment]
There are 2 lumps of steel in steel_standard already. Do you think it should use more?

**Comment by I-am-Erk** [comment]
Ah, I missed steel_standard. Carry on

**Pull request merged**